documentation/asciidoc/topics/yaml/publisher_labels_downstream.yaml

```diff
@@ -1,14 +1,11 @@
 # Defines global labels for services.
 - name: INFINISPAN_OPERATOR_TARGET_LABELS
```

Comment:
Does it makes sense to provide operator env labels for downstream? User can't reasonable modify them. (User would have to set them everytime he upgrades in OLM deployments)

Reply:
@rigazilla WDYT? Sounds like maybe it should be community only?

Reply:
@oraNod actually we're the only publisher for downstream, so it's ok to not show this info to the customer.
If we want to go this way I would remove the whole chapter "Global labels for " downstream.

Reply:
@rigazilla and @Crumby thanks, guys. I've removed the global labels section from downstream.